components/email-sender/org.wso2.carbon.email.sender.core/src/main/java/org/wso2/carbon/email/sender/core/RegistryBasedResourceLoader.java

```diff
@@ -46,12 +46,12 @@ public InputStream getResourceStream(String name) throws ResourceNotFoundExcepti
             if (registry == null) {
                 throw new IllegalStateException("No valid registry instance is attached to the current carbon context");
             }
-            if (!registry.resourceExists(EMAIL_CONFIG_BASE_LOCATION + "/" + name + ".vm")) {
+            if (!registry.resourceExists(EMAIL_CONFIG_BASE_LOCATION + "/" + name)) {
                 throw new ResourceNotFoundException("Resource '" + name + "' does not exist");
             }
             org.wso2.carbon.registry.api.Resource resource =
-                    registry.get(EMAIL_CONFIG_BASE_LOCATION + "/" + name + ".vm");
-
+                    registry.get(EMAIL_CONFIG_BASE_LOCATION + "/" + name);
+            resource.setMediaType("text/plain");
```

Comment:
Ack, But we don't have any constant containing "text/plain" from imported classes to this bundle. So it is efficient to set it directly in here rather than importing any other class, just for this.